client/main.go

```diff
@@ -0,0 +1,90 @@
+package main
```

Comment:
So in general the convention is to create a directory of cmd.  in cmd separate it to a server and client.

Reply:
Done